kong/plugins/aws-lambda/schema.lua

```diff
@@ -14,5 +14,6 @@ return {
     log_type = {type = "string", required = true, default = "Tail", 
                        enum = {"Tail", "None"}},
     port = { type = "number", default = 443 },
+    unhandled_status = {type = "number"},
```

Comment:
Also, a validation function to check for `status > 99 and status <= 999` would be great here :)